src/Gauge/ThermalAssistantRenderer.cpp

```diff
@@ -113,6 +115,14 @@ ThermalAssistantRenderer::PaintRadarPlane(Canvas &canvas) const
                        +Layout::FastScale(small ? 2 : 4)),
                   p.At(-Layout::FastScale(small ? 2 : 4),
                        +Layout::FastScale(small ? 2 : 4)));
+
+  if (small)
+    return;
+
+  if (!circling.TurningLeft()) 
+    DrawCircleLabelVSpeed(canvas, p + PixelSize{-45, 0}, vario.average);
+  else 
+    DrawCircleLabelVSpeed(canvas, p + PixelSize{+45, 0}, vario.average);
```

Comment:
Why is this here? This is not described anywhere, your commit message doesn't mention it.

Reply:
lordfolken suggested it? i will refactor PR to include